src/App.js

```diff
@@ -1,31 +1,40 @@
 import './App.css';
-import React, { useState } from 'react';
+import React, { useState} from 'react';
 import { Route, Routes, Navigate } from 'react-router-dom'; 
 import LoginForm from './components/login';
 import ProjectList from './components/project-list';
+import UserContext from './components/user-context';
 
-function PrivateRoute({ element: Element, isLoggedIn }) { 
-  return isLoggedIn ? <Element /> : <Navigate to="/login" />;
+function PrivateRoute({ element: Element, isLoggedIn, token }) { 
+  return isLoggedIn ? <Element token={token} /> : <Navigate to="/login" />;
```

Comment:
Why are you passing down token? we have created context for it so we don't need to pass it down as prop.